javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/resolution/InstanceOfTest.java

```diff
@@ -551,6 +560,24 @@ public void test_shouldFail() {
 
     }
 
+    @Nested
```

Comment:
Test to ensure symbols in record patterns can be solved in instanceof expressions